cv_template/templates/cv_template/generated_cv.html

```diff
@@ -4,7 +4,13 @@
 {% block extra_css %}
 <style>
   .emphasis-color {
-    color: {{cv.color}};
+    color: {
+        {
+        cv.color
+      }
+    }
+
+    ;
   }
```

Comment:
Nitpick (Non-blocking): Looks like something happened with the formatting here.

Reply:
This is because the CSS formatting is not liking the Django template tag. When Django changes it on runtime it works fine. This actually wasn't something I put there, but I'm gonna find a different way to do this